javascript/javascript2/week2/index.js

```diff
@@ -0,0 +1,125 @@
+
+
+// 1. Movies with a short title 
+
+const shortMovies = movies.filter(short => short.title.length <= 3);
+console.log(shortMovies);
+
+
+
+// 2. Movies with long title
+
+const longMovies = movies.filter(long => long.title.length >=20 );
+console.log(longMovies);
+
+
+
+// 3. Count the number of movies between 1980-1989 including both 
+
+const count = movies.filter(title => {
+    return title.year >= 1980 && title.year <= 1989;
+}).length;
```

Comment:
Excellent solution. A side note: the boolean condition can be done in one *expression*, so you could also write `movies.filter(title => title.year >= 1980 && title.year <= 1989)`, but that's a minor detail

Reply:
Thanks for the explanation.